### PR TITLE
Adding solutions to the Euler problems

### PR DESCRIPTION
**Pull request by drenniks**
## Description
Here are my solutions to the Euler problems (1-5) for this week.

## Weekly Journal
I really enjoyed working on these problems. The problems I found the most difficult were the largest prime number problem (#3) and the division problem (#5). For problem three, I wasn't quite sure how to implement the solution into while statement, which is why I have a for loop checking that my results were right. I know this can be done in a better way. For problem five, I had a hard time efficiently running the code since I was looping through large numbers. My solution to this is much faster than just looping over a large range of numbers, but I am wondering if there is a better way to solve this.  

## Questions
I don't have a lot of questions, I'm mainly just looking for feedback on what I have done and am curious to see more efficient solutions.

### CODE REVIEW

File: Week01/Problem01/dskinner_01.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    num = []
+    for i in arr:
+        if (((i/a).is_integer()) == True) or (((i/b).is_integer()) == True):
+            num.append(int(i))
+        else:
+            continue
+    return 'The sum of all the multiples of ' + str(a) + ' and ' + str(b) + ' is ' + str(np.sum(num)) +'!'
+
+#
+# Alternatively, here is the direct solution which will print upon typing `python dskinner_01.py`
```
Comment (dsirianni) [inline Week01/Problem01/dskinner_01.py]: If ever you wish to _only_ run a particular piece of code when a script is called directly, i.e., 
```
$ python myscript.py
```
on the command line (as you have intended here), you can place the code behind a special conditional statement:
```
if __name__ == "__main__":
   # This is only executed when script is run directly from command line
   code code code
```
The special variable `__name__` is set by Python when any script is executed; it is specifically set to `"__main__ "` when the script is run directly.  This is super useful when you're writing a module full of functions, and the only reason to run the script directly is for testing purposes.

Comment (drenniks) [reply Week01/Problem01/dskinner_01.py]: Good idea! I didn't know about this until last weeks lecture. I will begin to implement this.

File: Week01/Problem01/dskinner_01.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+#
+# Here is a solution to the general problem for multiples of 2 numbers (a, b) below a certain value. This can be imported to another script:
+#
+def multiples(a, b, value):
```
[the remark this answers is not in the document]
Comment (drenniks) [reply Week01/Problem01/dskinner_01.py]: Again, good idea. I will start to do this.

File: Week01/Problem01/dskinner_01.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        else:
+            continue
+    return 'The sum of all the multiples of ' + str(a) + ' and ' + str(b) + ' is ' + str(np.sum(num)) +'!'
+
+#
+# Alternatively, here is the direct solution which will print upon typing `python dskinner_01.py`
+#
+a = 3
+b = 5
+arr = np.arange(1000)
```
[the remark this answers is not in the document]
Comment (drenniks) [reply Week01/Problem01/dskinner_01.py]: Oh yes I can do that, I wasn't exactly sure how you were going to run it and I was playing around with functions. I will keep it simple.

**Comment by dsirianni** [comment]
Assigning @bencomer to look at your Week02 submission, since it got added to this PR.  I'm approving your Week 01 submission, so if Ben approves, he can go ahead and merge!

**Pull request merged**